Development

tati-frog opened the pull request:
Basicamente se mejoró la lectura del codigo. Aún no terminé por completo, pero seguramente envie los proximos cambios en otro PR. AVISO: Dejé algunas notas en los codigos con este formato ` // TODO: x ` para avisar que no sabia para que servia X cosa pero que se puede mejorar haciendo x cosa.

Commit:
Se solucionaron algunos errores en las dependecias

Commit:
Added package-lock.json to gitignore

Commit:
Codigo de index.js mas bonito

Commit:
Codigo de index.js mas bonito

Commit:
Se resolvieron conflictos

Commit:
Se creó la clase padre Event y se implementó la herencia en los eventos

Commit:
Se mejoró el codigo de los eventos de discord

Commit:
Se mejoró el codigo de los eventos de dbl

Commit:
solucionado bug

estebanDT30 commented:
Como nota, todos los commits que hagas en la rama, serán automáticamente agregados a este PR.

tati1206 commented:
nice

wwmon commented:
sexo

estebanDT30 committed:
Formateo y limpieza de código.

Recomiendo fuertemente usar Prettier para formatear el código.
https://prettier.io

estebanDT30 commented:
He realizado un **commit** en el fork de @tati1206, en el que básicamente he optimizado el código a como lo venimos haciendo en el proyecto.

tati1206 commented:
Ah, no me habia fijado del prettier y el jshint. xD
